### PR TITLE
feat: upgrade oa verify

### PR DESCRIPTION
**Pull request by simboonlong**
- bumping up version, related to https://github.com/Open-Attestation/oa-verify/pull/121

### CODE REVIEW

**Review by Nebulis — APPROVED** [review]

File: src/verify.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const registryVerifier: Verifier<
       const documentData = getData(document);
       return documentData.proof.method === v3.Method.DocumentStore;
     }
-    const documentData = getData(document);
+    const documentData = getData(document as WrappedDocument<v2.OpenAttestationDocument>);
```
Comment (Nebulis) [inline src/verify.ts:86]: why ?

Comment (simboonlong) [reply src/verify.ts:86]: there's build error: https://app.circleci.com/pipelines/github/OpenCerts/verify/66/workflows/e23b03ce-4272-4df6-931e-887735406e1e/jobs/64

Comment (Nebulis) [reply src/verify.ts:86]: ok but that's not good. The purpose of using DeepPartial is to make sure we do the correct checking here, however by casting to `WrappedDocument<v2.OpenAttestationDocument>` we just get rid of that purpose.

I'm not sure how to fix this now

Comment (Nebulis) [reply src/verify.ts:86]: this is the best I could come up with https://github.com/Open-Attestation/oa-verify/blob/master/src/verifiers/documentStoreStatus/openAttestationEthereumDocumentStoreStatus.ts#L52

let's use sth like this for the moment

File: src/verify.ts [hunk 2]
```diff
@@ -83,8 +83,13 @@ export const registryVerifier: Verifier<
       const documentData = getData(document);
       return documentData.proof.method === v3.Method.DocumentStore;
     }
-    const documentData = getData(document as WrappedDocument<v2.OpenAttestationDocument>);
-    return documentData.issuers.some(issuer => "documentStore" in issuer || "certificateStore" in issuer);
+
+    if (utils.isWrappedV2Document(document)) {
```
Comment (Nebulis) [inline src/verify.ts:87]: dont use this one because it actually doesn't test anything and I dunno how to write one that is actually helpful

**Review by Nebulis — APPROVED** [review]

**Pull request merged**